gulpfile.js

```diff
@@ -76,17 +81,24 @@ gulp.task('startWatcher', function(){
   gulp.watch(cssFiles, ['css']);
 });
 
-gulp.task('remove-build-zip', function(callback){
-  del.sync([buildDest + "/" + name + '.zip'], { force: true });
-  callback();
+gulp.task('remove-build-folder', function(){
+  return del([buildDest], { force: true });
 });
 
 gulp.task('zip-build', function(){
   return gulp.src(buildDest + '/**/*')
-    .pipe(zip(name + '.zip'))
+    .pipe(zip(name + '_' + version + '.zip'))
     .pipe(gulp.dest(buildDest));
 });
 
+gulp.task('update-qext-version', function () {
+  return gulp.src("./build/" + name + ".qext")
```

Comment:
need to use buildDestination from settings for it to work when BUILD_PATH is set